docs/src/modules/branding/BrandingRoot.tsx

```diff
@@ -208,14 +208,14 @@ theme = createMuiTheme(theme, {
             boxShadow: `0 0 0 0.25rem ${alpha(theme.palette.greyD7, 0.5)}`,
           },
           '&:hover': {
-            backgroundColor: darken(theme.palette.greyD7, 0.1),
+            backgroundColor: darken(theme.palette.greyD7, 0.07),
```

Comment:
"reduce lag feeling"

Uh, how?

Reply:
Well, if you don't feel the difference, then it's a false attempt. I still do hope that the new opacities look better. 

Oh no actually GitHub do 100ms for the hover in and 200m for the hover out transitions. Interesting tradeoff, but maybe too hard to customize and feel "unbalanced".

Reply:
> Well, if you don't feel the difference, then it's a false attempt.

I didn't compare them side by side, but before looking at the code I couldn't notice a difference. And since changing the opacity doesn't change the transition duration, I'm not surprised. 🙂